### PR TITLE
This commit fixes #43089

### PR DESCRIPTION
**Pull request by guptadhaval18**
Fixed 2 non-working URLs

### CODE REVIEW

File: site/en/guide/tf_numpy.ipynb [hunk 1]
```diff
@@ -581,7 +581,7 @@
       "source": [
         "### Trace compilation: tf.function\n",
         "\n",
-        "Tensorflow's `tf.function` works by \"trace compiling\" the code and then optimizing these traces for much faster performance. See the [Introduction to Graphs and Functions](./guide/intro_to_graphs).\n",
+        "Tensorflow's `tf.function` works by \"trace compiling\" the code and then optimizing these traces for much faster performance. See the [Introduction to Graphs and Functions](https://www.tensorflow.org/guide/intro_to_graphs).\n",
```
Comment (lamberta) [inline site/en/guide/tf_numpy.ipynb:584]:
```suggestion
        "Tensorflow's `tf.function` works by \"trace compiling\" the code and then optimizing these traces for much faster performance. See the [Introduction to Graphs and Functions](./intro_to_graphs.ipynb).\n",
```

File: site/en/guide/tf_numpy.ipynb [hunk 2]
```diff
@@ -787,7 +787,7 @@
       "source": [
         "## Performance comparisons\n",
         "\n",
-        "TensorFlow NumPy uses highly optimized TensorFlow kernels that can be dispatched on CPUs, GPUs and TPUs. TensorFlow also performs many compiler optimizations, like operation fusion, which translate to performance and memory improvements. See [TensorFlow graph optimization with Grappler](./guide/graph_optimization) to learn more.\n",
+        "TensorFlow NumPy uses highly optimized TensorFlow kernels that can be dispatched on CPUs, GPUs and TPUs. TensorFlow also performs many compiler optimizations, like operation fusion, which translate to performance and memory improvements. See [TensorFlow graph optimization with Grappler](https://www.tensorflow.org/guide/graph_optimization) to learn more.\n",
```
Comment (lamberta) [inline site/en/guide/tf_numpy.ipynb:790]:
```suggestion
        "TensorFlow NumPy uses highly optimized TensorFlow kernels that can be dispatched on CPUs, GPUs and TPUs. TensorFlow also performs many compiler optimizations, like operation fusion, which translate to performance and memory improvements. See [TensorFlow graph optimization with Grappler](./graph_optimization.ipynb) to learn more.\n",
```

**Review by lamberta — CHANGES_REQUESTED** [review]
Thanks for the fix. If you use a relative path and add the file extension then the link will work in GitHub and the website

**Comment by guptadhaval18** [comment]
@lamberta thanks for help. This is my first open source pull request.

**Review by lamberta — APPROVED** [review]
Thanks for the contribution and welcome!

**Pull request merged**